bee-runtime/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "bee-runtime"
+version = "0.1.0-alpha"
+authors = ["IOTA Stiftung"]
+edition = "2018"
+description = "Runtime components and utilities for the bee node framework"
+readme = "README.md"
+repository = "https://github.com/iotaledger/bee"
+license = "Apache-2.0"
+keywords = ["iota", "tangle", "bee", "framework", "runtime"]
+homepage = "https://www.iota.org"
+
+[dependencies]
+bee-storage = { git = "https://github.com/iotaledger/bee.git", branch = "dev" }
```

Comment:
This probably needs to be versioned to release it ?

Reply:
Yes, true